routes/login.js

```diff
@@ -24,6 +24,7 @@ router.post('/', async(req, res) => {
 			email: user.email,
 			role: user.role,
 			token: `Bearer ${token}`,
+			expiresIn: 1 * 60 * 60 * 1000,
```

Comment:
Just hardcoded the expiration period. couldnt get the exact time in ms when it will be expired from jwt library. If could, I shouldn't calculate it by hand to store this time into localStorage, because it will be different time.

for example: BE generates token at 1650831741, but FE calculates it as 1650831800 and stores into localStorage (it means to fetch data from BE it took about 69ms)